defaults/main.yml

```diff
@@ -13,5 +13,17 @@ letsencrypt_certbot_verbose: true
 letsencrypt_pause_services: []
 # Force Certificate Reneval
 letsencrypt_force_renew: true
-# Additional certbot arguments
+# certbot custom arguments -- see: https://certbot.eff.org/docs/using.html
 letsencrypt_certbot_args: []
+letsencrypt_certbot_combined_args: []
```

Comment:
This shouldn't be here afaict. `_letsencrypt_certbot_combined_args` is populated automatically later in the tasks.